lib/tasks/bundlers/generateFlexChangesBundle.js

```diff
@@ -39,6 +43,12 @@ module.exports = function({workspace, options}) {
 			log.verbose("sap.ui.fl not found in manifest.json, inserting it...");
 			mLibs["sap.ui.fl"] = {};
 		}
+
+		// if there are ctrl variant and minUI5Version < 1.72 need to update minUI5Version to 1.72.0
+		if (hasVariant && !hasFlexBundleVersion) {
+			log.verbose("minUI5Version < 1.72, updated it to 1.72.0");
+			data["sap.ui5"].dependencies.minUI5Version = "1.72.0";
```

Comment:
IMHO, the tooling must not change manifest metadata on the fly, it should rather stop with an error.

Reply:
Thanks, I change the logic.
The build will stop when the minUI5Version is below 1.72 and there are control variants in the change folder